airqo-platform/AirQo-frontend
# Ft registry update

What does this PR do?
- Updates action dialogs in device registry

- Which JIRA card(s)?
  N/A
- How do I test out this PR?
  Fetch this branch, cd to airqo-frontend/locate, run nom install and then npm run local. Device registry service and Location regstry service should be running in the backend
